data/meson.build

```
@@ -1,12 +1,12 @@
 install_data(
     'autostart.desktop',
-    install_dir: join_paths(get_option('datadir'), 'applications'),
+    install_dir: join_paths(get_option('prefix'), get_option('datadir'), 'applications'),
```

looks like prefix and datadir are already defined variables?
```suggestion
    install_dir: prefix / data_dir / 'applications',
```